docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -511,3 +511,10 @@ installed integrations.
 
 . Under advanced settings, set `windows.advanced.diagnostic.enabled`
 to `false`, and then save the integration.
+
+
+[discrete]
+[[hosted-agent-offline]]
+== Hosted {agent} Offline
+
+In Elastic Cloud on scaling up or down the hosted {agent}, new containers for the Elastic Agent are started and the old ones are shut down. The old Elastic Agents will show up in the list for 24 hours and will automatically disappear afterwards.
```

Comment:
```suggestion
To scale the {fleet-server} deployment, {ecloud} starts new containers or shuts down old ones when hosted {agent}s are required or no longer needed. The old {agent}s will show in the Agents list for 24 hours then automatically disappear.
```

Reply:
@ruflin I think it's useful to make the connection with Fleet Server here, but not sure if I have the technical details quite right.

Reply:
Make sense to add a notion of fleet-server here to better clarify it. We should make sure we use the same terms / explanations across the docs for it.